db/re/mob_db.conf

```diff
@@ -82288,15 +82288,420 @@ mob_db: (
-//3015,EP14_3_DEATH_B_MOB3
-//3016,EP14_3_DEATH_C_MOB1
-//3017,EP14_3_DEATH_C_MOB2
-//3018,EP14_3_DEATH_C_MOB3
+{
+	Id: 3010
+	SpriteName: "EP14_3_DEATH_A_MOB1"
+	Name: "Corrupt Orc Baby"
+	Lv: 158
+	Hp: 250000
+	Sp: 1
+	Exp: 12390
+	JExp: 16104
+	AttackRange: 1
+	Attack: [1036, 1415]
+	Def: 240
+	Mdef: 50
+	Stats: {
+		Str: 120
+		Agi: 85
+		Vit: 80
+		Int: 60
+		Dex: 88
+		Luk: 50
+	}
+	ViewRange: 10
+	ChaseRange: 12
+	Size: "Size_Small"
+	Race: "RC_DemiHuman"
+	Element: ("Ele_Earth", 2)
+	Mode: {
+		CanMove: true
+		Aggressive: true
+		CanAttack: true
+	}
+	MoveSpeed: 150
+	AttackDelay: 864
+	AttackMotion: 400
+	DamageMotion: 150
+	MvpExp: 0
+	Drops: {
+		Yggdrasilberry: 10
+		Old_Blue_Box: 20
+		Branch_Of_Dead_Tree: 20
+		Yellow_Gemstone: 20
+		Blue_Gemstone: 20
+		Red_Gemstone: 20
+		Fruit_Of_Mastela: 20
+	}
+},
+{
+	Id: 3011
+	SpriteName: "EP14_3_DEATH_A_MOB2"
+	Name: "Corrupt Baby Desert Wol"
+	Lv: 158
+	Hp: 232890
+	Sp: 1
+	Exp: 12390
+	JExp: 16104
+	AttackRange: 1
+	Attack: [1016, 1395]
+	Def: 240
+	Mdef: 45
+	Stats: {
+		Str: 100
+		Agi: 85
+		Vit: 100
+		Int: 88
+		Dex: 120
+		Luk: 50
+	}
+	ViewRange: 10
+	ChaseRange: 12
+	Size: "Size_Small"
+	Race: "RC_Brute"
+	Element: ("Ele_Fire", 2)
+	Mode: {
+		CanMove: true
+		Aggressive: true
+		CanAttack: true
+	}
+	MoveSpeed: 150
+	AttackDelay: 864
+	AttackMotion: 400
+	DamageMotion: 150
+	MvpExp: 0
+	Drops: {
+		Yggdrasilberry: 10
+		Old_Blue_Box: 20
+		Branch_Of_Dead_Tree: 20
+		Yellow_Gemstone: 20
+		Blue_Gemstone: 20
+		Red_Gemstone: 20
+		Fruit_Of_Mastela: 20
+	}
+},
+{
+	Id: 3012
+	SpriteName: "EP14_3_DEATH_A_MOB3"
+	Name: "Corrupt Familiar"
+	Lv: 158
+	Hp: 222550
+	Sp: 1
+	Exp: 12390
+	JExp: 16104
+	AttackRange: 1
+	Attack: [1002, 1381]
+	Def: 240
+	Mdef: 70
+	Stats: {
+		Str: 86
+		Agi: 85
+		Vit: 75
+		Int: 53
+		Dex: 100
+		Luk: 50
+	}
+	ViewRange: 10
+	ChaseRange: 12
+	Size: "Size_Small"
+	Race: "RC_Fish"
+	Element: ("Ele_Water", 2)
+	Mode: {
+		CanMove: true
+		Aggressive: true
+		CanAttack: true
+	}
+	MoveSpeed: 120
+	AttackDelay: 864
+	AttackMotion: 400
+	DamageMotion: 150
+	MvpExp: 0
+	Drops: {
+		Yggdrasilberry: 10
+		Old_Blue_Box: 20
+		Branch_Of_Dead_Tree: 20
+		Yellow_Gemstone: 20
+		Blue_Gemstone: 20
+		Red_Gemstone: 20
+		Fruit_Of_Mastela: 20
+	}
+},
+{
+	Id: 3013
+	SpriteName: "EP14_3_DEATH_B_MOB1"
+	Name: "Corrupt Orc Warrior"
+	Lv: 158
+	Hp: 300000
+	Sp: 1
+	Exp: 12390
+	JExp: 16104
+	AttackRange: 1
+	Attack: [1066, 1445]
+	Def: 240
+	Mdef: 60
+	Stats: {
+		Str: 150
+		Agi: 85
+		Vit: 150
+		Int: 40
+		Dex: 122
+		Luk: 70
+	}
+	ViewRange: 10
+	ChaseRange: 12
+	Size: "Size_Medium"
+	Race: "RC_DemiHuman"
+	Element: ("Ele_Earth", 2)
+	Mode: {
+		CanMove: true
+		Aggressive: true
+		CanAttack: true
+	}
+	MoveSpeed: 150
+	AttackDelay: 864
+	AttackMotion: 400
+	DamageMotion: 150
+	MvpExp: 0
+	Drops: {
+		Yggdrasilberry: 10
+		Old_Blue_Box: 20
+		Branch_Of_Dead_Tree: 20
+		Yellow_Gemstone: 20
+		Blue_Gemstone: 20
+		Red_Gemstone: 20
+		Fruit_Of_Mastela: 20
+	}
```

Comment:
Same questiosn for attack, mode (AI = 19 here) and card

AI 19:
No. | Bits   | Mob Type | Aegis/eA Description
--: | :----: | :------: | :----------------
 19 | 0x3095 | Any      | aggressive, change-target melee/chase, cast sensor idle


note: I did not check mobs 3014 onward, as I think these points would repeat.

Reply:
Done